TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/api/clusters.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from fastapi import HTTPException, APIRouter
+
+from console_link.api.sessions import http_safe_find_session
+from console_link.models.cluster import BasicAuth, BasicAuthArn, Cluster, ClusterInfo, NoAuth, SigV4Auth
+
+logging.basicConfig(format='%(asctime)s [%(levelname)s] %(message)s', level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+clusters_router = APIRouter(
+    prefix="/clusters",
+    tags=["clusters"],
+)
+
+
+def convert_cluster_to_api_model(cluster: Cluster) -> ClusterInfo:
+    """
+    Converts an internal Cluster model to the API ClusterInfo model.
+    """
+    if not cluster:
+        raise HTTPException(status_code=404, detail="Cluster not found")
+    
+    # Extract protocol from endpoint
+    protocol = "https" if cluster.endpoint.startswith("https://") else "http"
+    
+    if cluster.auth_type and cluster.auth_type.name == "BASIC_AUTH":
+        if cluster.auth_details and "user_secret_arn" in cluster.auth_details:
+            auth_details = cluster.auth_details["user_secret_arn"]
```

Comment:
Looks like a bug, we get `user_secret_arn` and then in the next line we get it again